opentech/apply/review/models.py

```diff
@@ -168,3 +168,19 @@ def update_submission_reviewers_list(sender, **kwargs):
         submission=review.submission,
         reviewer=review.author,
     )
+
+
+class ReviewOpinion(models.Model):
+    review = models.ForeignKey(Review, on_delete=models.CASCADE, related_name='opinions')
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.PROTECT,
+    )
+    opinion = models.IntegerField(choices=OPINION_CHOICES)
+
+    class Meta:
+        unique_together = ('author', 'review')
+
+    def get_author_role(self):
+        role = self.review.submission.assigned.with_roles().filter(reviewer=self.author).first()
```

Comment:
this is `assignment` not `role`.

Reply:
I think you're just talking about renaming this method? I'll do that.